apps/theming-designer/src/components/Samples.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import { SamplesCardHeader } from './SamplesCardHeader';
+import {
+  ActionButton,
+  Checkbox,
+  DefaultButton,
+  Dropdown,
+  Link,
+  PrimaryButton,
+  Stack,
+  Text,
+  TextField,
+  Toggle
+} from '../../../../packages/office-ui-fabric-react/lib';
```

Comment:
Instead of relative path imports, you should be able to use just `office-ui-fabric-react` (or `office-ui-fabric-react/lib/whatever`). If that's causing compiler errors, paste the error here and we'll figure out how to fix it.

Reply:
I tried to do: 
`import {
  ActionButton,
  Checkbox,
  DefaultButton,
  Dropdown,
  Link,
  PrimaryButton,
  Stack,
  Text,
  TextField,
  Toggle
} from 'office-ui-fabric-react/lib';`

but I'm getting the following error and looks like I'm not the only one because when I googled this I found that it happen in [other parts](https://github.com/OfficeDev/office-ui-fabric-react/issues/8146) of the Fabric code too. 
![image](https://user-images.githubusercontent.com/4161791/55022231-61354000-4fb7-11e9-8a45-643111b74ad1.png)

Reply:
Remove the `lib` part and it should work. Either that, or add an `index` segment after `lib`.

Reply:
Actually I take that back...just add an `index` segment.